src/sema/unused_variable.rs

```diff
@@ -481,10 +475,9 @@ pub fn check_unused_events(ns: &mut Namespace) {
                 }
             }
 
-            ns.diagnostics.push(generate_unused_warning(
+            ns.diagnostics.push(Diagnostic::warning(
                 event.loc,
-                &format!("event '{}' has never been emitted", event.name),
-                vec![],
+                format!("event '{}' has never been emitted", event.name),
```

Comment:
ideally quoting here should be fixed as well

Reply:
I've changed all the occurrences of `'{}'` in the file to ``` ´{}´ ```